### PR TITLE
Update issue templates

### PR DESCRIPTION
**Pull request by mitankit**

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+
+**Logs**
+If applicable, add logs from console to help explain your problem.
+
+**Screenshots**
+If applicable, add screenshots to help explain your problem.
+
+**Device Info (please complete the following information):**
+ - Device: [e.g. iPhone6]
+ - OS: [e.g. iOS8.1]
```
Comment (hokyungh) [inline .github/ISSUE_TEMPLATE/bug_report.md]: Wouldn't it be good idea to also put MediaLibrary version and SDK version?

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 2]
```diff
@@ -0,0 +1,34 @@
+---
```
Comment (tianyux-amz) [inline .github/ISSUE_TEMPLATE/bug_report.md]: Should we also include Media/SDK version in the bug report template?

File: .github/ISSUE_TEMPLATE/bug_report.md (hunk 1 above)
Comment (mitankit) [reply .github/ISSUE_TEMPLATE/bug_report.md]: Yes Good idea. Let me add it

File: .github/ISSUE_TEMPLATE/bug_report.md (hunk 2 above)
Comment (mitankit) [reply .github/ISSUE_TEMPLATE/bug_report.md]: Yes I will add it

**Pull request merged**